lib/marginalia.rb

```diff
@@ -67,33 +67,39 @@ def annotate_sql(sql)
       sql
     end
 
-    def execute_with_marginalia(sql, name = nil)
-      execute_without_marginalia(annotate_sql(sql), name)
+    def execute_with_marginalia(sql, *args)
+      execute_without_marginalia(annotate_sql(sql), *args)
     end
+    ruby2_keywords :execute_with_marginalia if respond_to?(:ruby2_keywords, true)
 
-    def exec_query_with_marginalia(sql, name = 'SQL', binds = [])
-      exec_query_without_marginalia(annotate_sql(sql), name, binds)
+    def exec_query_with_marginalia(sql, *args)
+      exec_query_without_marginalia(annotate_sql(sql), *args)
     end
+    ruby2_keywords :exec_query_with_marginalia if respond_to?(:ruby2_keywords, true)
 
     if ActiveRecord::VERSION::MAJOR >= 5
-      def exec_query_with_marginalia(sql, name = 'SQL', binds = [], **options)
+      def exec_query_with_marginalia(sql, *args, **options)
         options[:prepare] ||= false
-        exec_query_without_marginalia(annotate_sql(sql), name, binds, **options)
+        exec_query_without_marginalia(annotate_sql(sql), *args, **options)
       end
+      ruby2_keywords :exec_query_with_marginalia if respond_to?(:ruby2_keywords, true)
```

Comment:
Seeing this warning here: 
`lib/marginalia.rb:85: warning: Skipping set of ruby2_keywords flag for exec_query_with_marginalia (method accepts keywords or method does not accept argument splat)`

Reply:
Indeed. Good catch.